backend/application/syson-sysml-import/src/test/resources/ASTTransformerTest/convertNamespaceImportTest/model.sysml

```diff
@@ -0,0 +1,3 @@
+package Packets {
+    import ScalarValues;
```

Comment:
Could you please specify the visibility of the import?

Reply:
fait.